jdi-light-material-ui-tests/src/test/java/io/github/epam/material/tests/utils/TransitionTests.java

```diff
@@ -1,32 +1,51 @@
 package io.github.epam.material.tests.utils;
 
 import io.github.epam.TestsInit;
+import org.testng.annotations.BeforeMethod;
 import org.testng.annotations.Test;
 
-import static io.github.com.StaticSite.transitionFrame;
-import static io.github.com.StaticSite.utilsTransitionCollapsePage;
-import static io.github.com.StaticSite.utilsTransitionFadePage;
+import static io.github.com.MaterialNavigator.openSection;
+import static io.github.com.pages.utils.TransitionPage.checkboxes;
+import static io.github.com.pages.utils.TransitionPage.transitions;
+
+/**
+ * To see an example of Button web element please visit https://material-ui.com/components/transitions/
+ */
 
 public class TransitionTests extends TestsInit {
 
-    @Test
+    @BeforeMethod
+    public void before() {
+        openSection("Transitions");
+    }
+
+/*
+//both of this test were previously used
+  @Test
     public void fadeTransitionTest() {
-        utilsTransitionFadePage.open();
 
-        transitionFrame.transition.is().hidden();
-        transitionFrame.transition.clickSwitch();
-        transitionFrame.transition.is().visible();
-        transitionFrame.transition.has().content("Content");
+        transition.is().hidden();
+        transition.clickSwitch();
+        transition.is().visible();
+        transition.has().content("Content");
     }
 
     @Test
     public void collapseTransitionTest() {
-        utilsTransitionCollapsePage.open();
 
-        transitionFrame.transition.is().hidden();
-        transitionFrame.transition.clickSwitch();
-        transitionFrame.transition.is().visible();
-        transitionFrame.transition.has().content("Content");
-    }
+        transition.is().hidden();
+        transition.clickSwitch();
+        transition.is().visible();
+        transition.has().content("Content");
+    }*/
 
+    //tried to check functionality of elements but checkboxes does not work
+    @Test
+    public void transitionsTest() {
+        transitions.get(1).isEnabled();
```

Comment:
this construction not checking isEnabled, it's only return Boolean value
you need use is().enabled()